crates/sui-types/src/lib.rs

```diff
@@ -128,12 +134,20 @@ pub const SUI_RANDOMNESS_STATE_OBJECT_ID: ObjectID =
 pub const SUI_DENY_LIST_ADDRESS: AccountAddress = deny_list_addr();
 pub const SUI_DENY_LIST_OBJECT_ID: ObjectID = ObjectID::from_address(SUI_DENY_LIST_ADDRESS);
 
+/// 0x9: hardcode object ID for the singleton bridge object.
+pub const SUI_BRIDGE_ADDRESS: AccountAddress = address_from_single_byte(9);
+pub const SUI_BRIDGE_OBJECT_ID: ObjectID = ObjectID::from_address(SUI_BRIDGE_ADDRESS);
```

Comment:
I think we will need some more checks on the bridge object:
- Can we take it mutably? 
  - If no, we will need to duplicate the check like we have for other system objects 
  - If yes, make sure it doesn't have `store` so no one can wrap it or steal it

Reply:
the Bridge object is mutable, it is versioned and the wrapper object only have `key` ability